lesson07/assignment/cli3/src/views/Home.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+<div class="home">
+
+  <div id="app">
+
+    <button type="button" name="button" v-on:click="show = !show">Click for Tips!</button>
+  </div>
+
+  <div class="container">
+    <div v-if="show" class="container bg-warning mb-2">
```

Comment:
For the v-else, you can just add an else div that shows up a different message (content coming soon or whatever)